test/Interop/Cxx/class/nonescapable-lifetimebound.swift

```diff
@@ -0,0 +1,63 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: %target-swift-frontend -typecheck -I %swift_src_root/lib/ClangImporter/SwiftBridging  -I %t/Inputs  %t/test.swift -enable-experimental-feature NonescapableTypes -cxx-interoperability-mode=default -diagnostic-style llvm 2>&1
+
+//--- Inputs/module.modulemap
+module Test {
+    header "nonescapable.h"
+    requires cplusplus
+}
+
+//--- Inputs/nonescapable.h
+#include "swift/bridging"
+
+struct SWIFT_NONESCAPABLE View {
+    View() : member(nullptr) {}
+    View(const int *p [[clang::lifetimebound]]) : member(p) {}
+    View(const View&) = default;
```

Comment:
Could you please add one more test where the copy constructor has an annotated parameter (assuming Clang allows that)?

I don't think that would be reasonable in C++, and it's probably fine to drop that annotation in Swift, just wanted to make sure we don't crash or miscompile.

Reply:
Added a test.